### ISSUE TITLE
NoMethodError: private method `eval' called for nil:NilClass on Ruby 2.2 and Ubuntu12.04

### ISSUE THREAD

**Issue by sanemat** [k-tsj/power_assert#10]
I use power-assert and test-unit, and I use stub library (mocha or test-unit-rr), then I met error below:

```
Error: test: #remote_urls(Saddler::Reporter::Support::Git::TestGitRepository)
: NoMethodError: private method `eval' called for nil:NilClass
/home/travis/.rvm/gems/ruby-2.2.0@global/gems/power_assert-0.2.2/lib/power_assert.rb:116:in `block in initialize'
```

I don't have minimal reproduce code yet, do you know what this problem is?
This happens only Ruby 2.2 and Ubuntu12.04. Ruby2.1 and Ubuntu 12.04, Ruby 2.2 and mac os x 10.9.5 works fine.

my env:
power_assert (0.2.2)
test-unit (3.0.9)
test-unit-rr (1.0.3)

This is build:
https://travis-ci.org/packsaddle/ruby-saddler-reporter-support-git/builds/52102529
https://travis-ci.org/packsaddle/ruby-saddler-reporter-support-git/jobs/52102533

This is code:
test-unit, power-assert and mocha
https://github.com/packsaddle/ruby-saddler-reporter-support-git/pull/3
test-unit, power-assert and test-unit-rr
https://github.com/packsaddle/ruby-saddler-reporter-support-git/pull/2

**Comment by k-tsj** [comment]
It is a bug of ruby introduced at r47670 and fixed at r49430.

I've just released power_assert 0.2.3, including workaround for the issue.
Please try it.

**Comment by sanemat** [comment]
@k-tsj Thanks! 0.2.3 works fine! Great!
https://travis-ci.org/packsaddle/ruby-saddler-reporter-support-git/jobs/52599284
